src/views/shared/confirmation-modal/confirmation-modal.styles.ts

```diff
@@ -4,12 +4,15 @@ import { Theme } from "src/styles/theme";
 
 export const useConfirmationModalStyles = createUseStyles((theme: Theme) => ({
```

Comment:
Under 330px the button is not properly rendered

<img width="296" alt="image" src="https://user-images.githubusercontent.com/15896177/229824255-3b308ba4-be58-4ef5-901d-e341dd3ba90b.png">

Reply:
No mobile phone has 330px width nowadays... I took as a reference a Pixel 5 (a mobile phone from 3 years ago) and an iPhone SE (released in 2016) to test it and it works fine on that mobiles. Same for the comment below.